### PR TITLE
Feature/12 completed section travel point

### PR DESCRIPTION
**Pull request by thuan734655**
completed main-section-travelPoint

### CODE REVIEW

File: src/Pages/index.html
Comment (DatWithTheWorld) [inline src/Pages/index.html:1]: end of line

File: src/stylesheet/Component/_secondTitle.scss
Comment (DatWithTheWorld) [inline src/stylesheet/Component/_secondTitle.scss:1]: same previous PR

Comment (thuan734655) [reply src/stylesheet/Component/_secondTitle.scss:1]: I fixed it. Thanks

File: src/Pages/index.html
Comment (thuan734655) [reply src/Pages/index.html:1]: I fixed it

**Pull request closed without merging**